content/docs/tasks/security/authn-policy/index.md

```diff
@@ -590,6 +591,9 @@ You also need the `key.pem` file:
 $ wget {{< github_file >}}/security/tools/jwt/samples/key.pem
 {{< /text >}}
 
+You may also need to download the [jwcrypto](https://pypi.org/project/jwcrypto) library,
```

Comment:
```suggestion
{{< tip >}}
Download the [jwcrypto](https://pypi.org/project/jwcrypto) library,
```

Reply:
Done. By the way, what's the most straightforward way to preview the markdown with all of the macros?